NodeJS/__integration__/RestClientAgent.it.ts

```diff
@@ -79,6 +81,11 @@ describe("RestClient", () => {
         console.log(submitted)
         expect(submitted).toBeDefined()
     })
+    test("getNextAvailableExtension for skill", async () => {
```

Comment:
Two things here. 

First the minor thing, these test descriptions should read like a sentence. Since your `describe` has "RestClient", I would recommend updating the text to be something like: "can get next available skill number for account"

Then the test output will look like:

```
RestClient
  /* other test output */
  can get next available skill number for account
    PASS
  /* other test output */
```

Second, I'm not sure that this test is all that useful. Because you're strongly typing your `RestClient`, you already know for certain that you will _always_ be returned a `Promise<number>`, and this test doesn't do anything else. If you were to have it check to ensure that it truly got the *next* number, that would make sense, but since this is a call to the API, that should be a Java test. So, if you just want to ensure that you have a high code coverage, then this isn't bad, but I would argue that this particular test is not adding additional confidence to the reliability of the product.

Reply:
The test is there to make sure you are firing a correct call to the backend.  It is to make sure in this case the url is correct.  The integration test is not used to improve code coverage. It is used to achieve correctness.  It is run manually unless there is a reproducible env one could run against.

Reply:
ah, I had assumed that this was a unit test. I did not realize that these were actually hitting the server. 

feels weird to me that you're using TS to do integration tests, but if it work then sweet!

Reply:
The project is about writing a typescript client against admin center backend.  Got to hit the real backend to get it going.